### ISSUE TITLE
apply to Docker consistency guideline

### ISSUE THREAD

**Issue by CCFenner** [SonarSource/sonar-scanner-cli-docker#30]
When running in a Jenkins environment the following issue is raised:

```log
ERROR: The container started but didn't run the expected command. Please double check your ENTRYPOINT does execute the command passed as docker run argument, as required by official docker images (see https://github.com/docker-library/official-images#consistency for entrypoint consistency requirements).
Alternatively you can force image entrypoint to be disabled by adding option `--entrypoint=''`.
```

Jenkins pipeline:
```groovy
node {
  docker.image('sonarsource/sonar-scanner-cli').inside{
    sh "sonar-scanner --version"
  }
}
```

see https://github.com/docker-library/official-images#consistency
